build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -235,13 +235,16 @@ class RuntimeTestConfig {
           ],
         },
       ];
+      const newBabelifyConfig = Object.assign({}, this.babelifyConfig);
 
-      this.browserify.transform = [
-        [
-          'babelify',
-          Object.assign({}, this.babelifyConfig, {plugins: [plugin]}),
-        ],
-      ];
+      // don't overwrite old plugins
+      if ('plugins' in newBabelifyConfig) {
```

Comment:
Same here, since we always set `plugin` to newBabelifyConfig in `karma.conf.js`, do we still need the check?